application.py

```diff
@@ -10,36 +11,36 @@
 from structlog.stdlib import LoggerFactory, add_log_level
 from structlog.threadlocal import wrap_dict
 
-EQ_LOG_LEVEL = os.getenv('EQ_LOG_LEVEL', 'INFO')
 EQ_WERKZEUG_LOG_LEVEL = os.getenv('EQ_WERKZEUG_LOG_LEVEL', 'INFO')
 EQ_DEVELOPER_LOGGING = os.getenv('EQ_DEVELOPER_LOGGING', 'False').upper() == 'TRUE'
 
 
 def configure_logging():
-    # set up some sane logging, as opposed to what flask does by default
-    log_format = "%(message)s"
-    levels = {
-        'CRITICAL': logging.CRITICAL,
-        'ERROR': logging.ERROR,
-        'WARNING': logging.WARNING,
-        'INFO': logging.INFO,
-        'DEBUG': logging.DEBUG,
-    }
-    handler = logging.StreamHandler()
+    log_level = logging.INFO
+    if os.getenv('FLASK_ENV') == 'development':
+        log_level = logging.DEBUG
+
+    info_handler = logging.StreamHandler(sys.stdout)
+    info_handler.setLevel(log_level)
+    info_handler.addFilter(lambda record: record.levelno <= logging.WARNING)
+
+    error_handler = logging.StreamHandler(sys.stderr)
+    error_handler.setLevel(logging.ERROR)
+
     logging.basicConfig(
-        level=levels[EQ_LOG_LEVEL], format=log_format, handlers=[handler]
+        level=log_level, format='%(message)s', handlers=[error_handler, info_handler]
     )
 
     # Set werkzeug logging level
     werkzeug_logger = logging.getLogger('werkzeug')
-    werkzeug_logger.setLevel(level=levels[EQ_WERKZEUG_LOG_LEVEL])
+    werkzeug_logger.setLevel(level=logging.getLevelName(EQ_WERKZEUG_LOG_LEVEL))
 
     def parse_exception(_, __, event_dict):
         if EQ_DEVELOPER_LOGGING:
```

Comment:
We could remove `EQ_DEVELOPER_LOGGING` and use `FLASK_ENV` to set this as we do for the log level.